less/index.less

```diff
@@ -55,40 +55,67 @@ table {
 
 html, body {
 	height: 100%;
-	background-color:darkgrey;
 }
+
+@background-color:lightgrey;
```

Comment:
I would use more descriptive variable names, a future coworker may scorn you for non descriptive variable names.